tests/integration/suite/daprd/jobs/http/basic.go

```diff
@@ -98,4 +103,51 @@ func (b *basic) Run(t *testing.T, ctx context.Context) {
 			assert.Empty(t, string(body))
 		}
 	})
+
+	t.Run("overwrite if exists", func(t *testing.T) {
+		type request struct {
+			name string
+			body string
+		}
+
+		for _, r := range []request{
+			{"overwrite1", `{"schedule": "@daily"}`},
+			{"overwrite1", `{"schedule": "@daily", "repeats": 3, "overwrite": true, "due_time": "10s", "ttl": "11s"}`},
+		} {
+			postURL := fmt.Sprintf("http://localhost:%d/v1.0-alpha1/jobs/%s", b.daprd.HTTPPort(), r.name)
+			req, err := http.NewRequestWithContext(ctx, http.MethodPost, postURL, strings.NewReader(r.body))
+			require.NoError(t, err)
+			resp, err := httpClient.Do(req)
+			require.NoError(t, err)
+			assert.Equal(t, http.StatusNoContent, resp.StatusCode)
+			body, err := io.ReadAll(resp.Body)
+			require.NoError(t, err)
+			require.NoError(t, resp.Body.Close())
+			assert.Empty(t, string(body))
+		}
```

Comment:
Again, please move to a new int test `tests/integration/suite/daprd/jobs/http/overwrite.go`.